Classes/Query.php

```diff
@@ -1193,16 +1193,13 @@ public function setHighlighting($highlighting = true, $fragmentSize = 200)
                 $this->queryParameters['hl.fl'] = $highlightingFields;
             }
 
-            $useFastVectorHighlighter = $this->solrConfiguration->getSearchResultsHighlightingUseFastVectorHighlighter();
-            if ($useFastVectorHighlighter) {
-                if ($fragmentSize <= 18) {
-                    throw new \InvalidArgumentException("The setting useFastVectorHighlighter can only be used with a fragementSize larger then 18");
-                }
-                $this->queryParameters['hl.useFastVectorHighlighter'] = 'true';
-            }
-
+            // the fast vector highlighter can only be used, when the fragmentSize is
+            // higher then 17 otherwise solr throws an exception
+            $useFastVectorHighlighter = $fragmentSize >= 18;
```

Comment:
Can you wrap this in braces please? `$useFastVectorHighlighter = ($fragmentSize >= 18);` Makes it more clear we're evaluating to a boolean.